### ISSUE TITLE
"export 'Container' was not found in '@vuex-orm/core'

### ISSUE THREAD

**Issue by gunarssimkuns** [vuex-orm/vuex-orm#470]
https://vuex-orm.github.io/vuex-orm/api/database/database.html#database

```js
import { Container } from '@vuex-orm/core'

const database = Container.database

const user = database.model('users')
```

webpack outputs: `"export 'Container' was not found in '@vuex-orm/core'`

I am using @vuex-orm/core@0.32.4

**Comment by kiaking** [comment]
@gunarssimkuns Oh damn. Thank you for pointing out! And, thanks for the PR too! I'll check the PR and I'll comment if there's anything.

**Comment by kiaking** [comment]
The issue has been fixed at Vuex ORM 0.32.5! Thank you so much for the report and with the awesome PR 🎉 
https://github.com/vuex-orm/vuex-orm/releases/tag/v0.32.5
